### PR TITLE
Add an endpoint called status/healthcheck/es

### PR DESCRIPTION
**Pull request by robbiethegeek**
this needs to be validated within the VA.

### CODE REVIEW

File: src/server/routes/status.js [hunk 1]
```diff
@@ -1,12 +1,78 @@
...
+const validIdForEnv = (env) => {
+  switch (env) {
+  // SQA
+  // 382363508
+    case 'development':
+      return '377609264';
+    case 'staging':
+      return '3623520284';
+    case 'production':
+      return '19831620752';
```
Comment (alexose) [inline src/server/routes/status.js]: What are these IDs exactly?  Should they be in config.js?

File: src/server/utils/tls.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const tlsUtils = (config) => {
+  return {
+    readTLSArtifacts: () => {
+      const artifacts = {};
+
+      // If either the client cert or key is set, try to read both and die hard if
+      // either fails. It's nonsensical to have one without the other and it's hard to
+      // debug a server that doesn't have a cert/key when you think it does so
+      // hard dying is good here on a configuration error.
```
Comment (alexose) [inline src/server/utils/tls.js]: Yeah, this seems sensible.

**Comment by alexose** [comment]
This looks good.  It succeeds on the hardwired VA connection:

```
AlexandersMBP2% curl http://localhost:3000/api/hca/v1/status/healthcheck/es -v
*   Trying ::1...
* Connected to localhost (::1) port 3000 (#0)
> GET /api/hca/v1/status/healthcheck/es HTTP/1.1
> Host: localhost:3000
> User-Agent: curl/7.43.0
> Accept: */*
>
< HTTP/1.1 200 OK
< X-Powered-By: Express
< Date: Wed, 08 Jun 2016 23:36:16 GMT
< Connection: keep-alive
< Content-Length: 0
<
```

And fails when I'm tethering on my phone:

```
AlexandersMBP2% curl http://localhost:3000/api/hca/v1/status/healthcheck/es -v
*   Trying ::1...
* Connected to localhost (::1) port 3000 (#0)
> GET /api/hca/v1/status/healthcheck/es HTTP/1.1
> Host: localhost:3000
> User-Agent: curl/7.43.0
> Accept: */*
>
< HTTP/1.1 500 Internal Server Error
< X-Powered-By: Express
< Date: Wed, 08 Jun 2016 23:36:00 GMT
< Connection: keep-alive
< Content-Length: 0
<
```

Which is the expected result.  👍

File: src/server.js [hunk 3]
```diff
@@ -1,10 +1,16 @@
...
 const express = require('express');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
 const winston = require('winston');
 
-const options = { logger: winston };
 const config = require('../config');
+
+const options = {
+  config,
```
Comment (alexose) [inline src/server.js]: Any particular reason why we're sharing config as a global on the application object?  (This is usually how I do it, just curious about the change)

Comment (robbiethegeek) [inline src/server.js]: Seemed silly for us to load a few times per app

**Comment by robbiethegeek** [comment]
Merging this in after addressing @alexose's comments / nits.

**Pull request merged**